### DISCUSSION TITLE
The declared package dependency does not exist. Dependency: Microsoft.WindowsAppSDK.1.0-experimental1. Minimum version: 0.218.840.0.

### DISCUSSION THREAD

**Discussion by ilovepumpkin** [microsoft/WindowsAppSDK#1708] [Q&A] [answered]
I am trying to publish a package (which is using Window App SDK experimental ) to Windows Store just for test. I got the error "The declared package dependency does not exist. Dependency: Microsoft.WindowsAppSDK.1.0-experimental1. Minimum version: 0.218.840.0.". Does it mean I can not publish the package unless it is using stable Windows App SDK? 

![image](https://user-images.githubusercontent.com/11310645/139781966-cd6225bc-31bd-4180-9890-68a104c534be.png)

**Comment by DrusTheAxe** [comment]
@BenJKuhn  @andrewleader tag

**Comment by BenJKuhn** [comment, marked as answer]
I love the enthusiasm, and your intuition is correct: Experimental versions of the Windows App SDK are intended for development purposes only, and are not available in the store, nor are they serviced. Stay tuned, the 1.0 release version is still on track for this quarter, per our roadmap here:

https://github.com/microsoft/WindowsAppSDK/blob/main/docs/roadmap.md

Thanks,

Ben

**Reply by ilovepumpkin** [reply]
Okay, thanks for your confirmation! So ... the 1.0 release date is not decided yet? So far, we only know it will be released in Q4 based on the roadmap. 

And ... if I use v0.8.5 to publish to store, there should not be the above error, correct?  https://github.com/microsoft/WindowsAppSDK/releases/tag/v0.8.5

**Comment by git102347501** [comment]
I encountered the same problem, which affected my enthusiasm for participation. Please provide the package version that can be put on the shelf as soon as possible

**Comment by BenJKuhn** [comment]
Windows App SDK 1.0 is available as of mid-November. If you're still developing against the experimental packages, you should update your Nuget reference to "Microsoft.WindowsAppSDK.1.0". You can find details about the release here:

https://github.com/microsoft/WindowsAppSDK/discussions/1806

Thanks,

Ben
